functable.c

```diff
@@ -206,6 +206,8 @@ static void init_functable(void) {
 #ifdef RISCV_RVV
     if (cf.riscv.has_rvv) {
         ft.adler32 = &adler32_rvv;
+        ft.chunkmemset_safe = &chunkmemset_safe_rvv;
+        ft.chunksize = &chunksize_rvv;
```

Comment:
Shouldn't `inflate_fast_rvv` be added here as well?

Reply:
Thanks, it's done. I ran benchmark with enabling `inflate_fast_rvv` here but I forgot to add this.